common/app/templates/inlineJS/blocking/loadFonts.scala.js

```diff
@@ -95,8 +100,21 @@ do you have fonts in localStorage?
                 })();
 
                 // use whatever font CSS we've now got
-                function useFont(el, css) {
+                function useFont(el, css, fontName) {
                     el.innerHTML = css;
+
+                    fontsToPost.push({
+                        fontName: fontName,
+                        css: css
+                    });
+
+                    // if all the fonts have loaded and postFonts true then post them to the parent
+                    if (postFonts && fontsToPost.length === fontsToLoadCount) {
+                        window.parent.postMessage({
+                            name:"guardianFonts",
+                            fonts: fontsToPost
+                        }, "*");
+                    }
```

Comment:
This code is being shipped (somewhat needlessly) in frontend as well as within `/font-loader` iframes. Do we want to wrap it in a Scala conditional to prevent it appearing on frontend? Or does this add too much complexity?

```scala
@if(postFonts) {
  fontsToPost.push({
    // ...
}
```